### ISSUE TITLE
error: failed to compile file: unknown opcode 11 in module wso2/twitter:0.9.23

### ISSUE THREAD

**Issue by 13Ax0** [wso2-ballerina/module-twitter#78]
**Description:**

I was reading the [Program Ballerina to Send a Tweet](https://ballerina.io/learn/quick-tour/#program-ballerina-to-send-a-tweet) section on [Quick Tour](https://ballerina.io/learn/quick-tour/) and copied the final code it provided. I got this:

```
$ ballerina run --config twitter.toml twttr.bal
error: failed to compile file: unknown opcode 11 in module wso2/twitter:0.9.23
```

I looked for this repository and found [another example in the README](https://github.com/wso2-ballerina/module-twitter#example). There were two errors (maybe I should open another issue about this?), so I tried fixing them. When everything was ok, I executed the program:

```
$ ballerina run twttr.bal
```

Yet again...

```
error: failed to compile file: unknown opcode 11 in module wso2/twitter:0.9.23
```

**Suggested Labels:**

I don't know if this is a bug, but I think is. I'm new to Ballerina (I started today).

**Affected Product Version:**

- wso2/twitter: 0.9.23
- Ballerina version: Ballerina 0.990.1

**OS, DB, other environment details and versions:** 

- OS: elementaryOS 0.5 Juno (Ubuntu-based)

**Steps to reproduce:**
 
- Copy the example provided in the README (as I said earlier, there are two errors, so I'll post the code I used here).

```ballerina
// File name: twttr.bal
import ballerina/io;
import wso2/twitter;

public function main() returns error? {
    twitter:Client twitterClient = new({
        clientId: "blah",
        clientSecret: "blah",
        accessToken: "blah",
        accessTokenSecret: "blah"
    });

    string status = "Twitter endpoint test";

    twitter:Status twitterStatus = check twitterClient->tweet(status);
    string tweetId = <string>twitterStatus.id;
    string text = twitterStatus.text;
    io:println("Tweet ID: " + tweetId);
    io:println("Tweet: " + text);
}
```

- Run `ballerina run twttr.bal`.

**Comment by MaryamZi** [comment]
@13Ax0 , this is an issue and we are currently working on the fix. Sorry about the inconvenience. We will get back to you with an update ASAP.

We've also opened a PR https://github.com/wso2-ballerina/module-twitter/pull/80 to fix the issues in the README file.

**Comment by 13Ax0** [comment]
@MaryamZi Thank you.

**Comment by MaryamZi** [comment]
@13Ax0 we've fixed this issue with Ballerina 0.990.2. 

Ballerina 0.990.2 is now available at https://ballerina.io/downloads.

Would it be possible to upgrade to Ballerina 0.990.2? Please let us know if you encounter further issues.

**Comment by 13Ax0** [comment]
@MaryamZi It works now. Thanks. I think we can close this issue.

**Comment by MaryamZi** [comment]
Thank you. Closing the issue since it is resolved with Ballerina 0.990.2.
